### PR TITLE
Xamarin Test App, MSAL minor fixes

### PR DESCRIPTION
**Pull request by SomkaPe**

### CODE REVIEW

File: dev apps/XForms/XForms/ArrayToStringConverter.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XForms
+{
+    internal class ArrayToStringConverter : IValueConverter
```
Comment (kpanwar) [inline dev apps/XForms/XForms/ArrayToStringConverter.cs]: there are already methods in MSAL that allow you to do ArrayToString converstion. see MsalStringHelper.cs

File: dev apps/XForms/XForms/CachePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class CachePage : ContentPage
+    {
+        public CachePage()
+        {
+            InitializeComponent();
+        }
+
+        private static string ToString(IEnumerable<string> collection)
```
Comment (kpanwar) [inline dev apps/XForms/XForms/CachePage.xaml.cs]: you can use msalstringhelper class here

File: dev apps/XForms/XForms/XForms.csproj [hunk 3]
```diff
@@ -1,18 +1,63 @@
...
-  </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="2.3.3.193" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\src\Microsoft.Identity.Client\Microsoft.Identity.Client.csproj" />
-  </ItemGroup>
-</Project>
\ No newline at end of file
+      <TargetFramework>netstandard1.1</TargetFramework>
```
Comment (kpanwar) [inline dev apps/XForms/XForms/XForms.csproj]: format this file.

File: src/Microsoft.Identity.Client/PublicClientApplication.cs [hunk 4]
```diff
@@ -64,6 +64,7 @@ public PublicClientApplication(string clientId, string authority)
             : base(authority, clientId, DEFAULT_REDIRECT_URI, true)
         {
             UserTokenCache = new TokenCache();
+            UserTokenCache.ClientId = clientId;
```
Comment (kpanwar) [inline src/Microsoft.Identity.Client/PublicClientApplication.cs]: use property initializers

File: dev apps/XForms/XForms/ArrayToStringConverter.cs (hunk 1 above)
Comment (SomkaPe) [inline dev apps/XForms/XForms/ArrayToStringConverter.cs]: done

**Pull request merged**